notebooks/data-sources/TestGrid/metrics/time_to_test.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
@MichaelClifford sure :+1:

Reply:
In order to address the above comment, there needs to be some changes in the way we are downloading the dataset and unrolling to a dataframe such that the "graph" parameter is also included in the process. 

So, instead of this notebook containing a copy of the data downloading and labelwise_encoding steps with minor edits, we can make those changes in the respective notebooks and keep this notebook only for metric calculation as is its original purpose. 

For addressing the above, I have opened a separate issue #171 , since the breadth of those steps span beyond the scope of time-duration calculation and need modifications in multiple places in the repo.

Reply:
@oindrillac :+1: